peps/pep-0758.rst

```diff
@@ -0,0 +1,432 @@
+a special process that can take some time to resolve, and the criteria for
+granting such exceptions isn't well documented.
+
+Reduce operational complexity
+=============================
+
+Standing up and maintaining an entire package index can be a complex
+operational solution, both time and resource intensive. This is especially
+true if the main purpose of such an index is just to avoid file size
+limitations. The external index approach also imposes a tricky UX on consumers
+of projects on the external index, requiring them to understand how CLI
+options such as ``--external-index-url`` work, along with the security
+implications of such flags. It would be much easier for both producers and
+consumers of large wheel packages to just stand up and maintain a simple web
+server, capable of serving individual files with no more complex API than
+``HTTP GET``. Such an interface is also easily cacheable or placed behind a
+`CDN <https://en.wikipedia.org/wiki/Content_delivery_network>`__. Simple HTTP
+servers are also much easier to audit for security purposes, easier to proxy,
+and usually take much less resources to run, support, and maintain.  Even
+something like `Amazon S3 <https://aws.amazon.com/s3/>`__ could be used to
+host external wheels.
+
+This PEP proposes an approach that favors such operational simplicity.
+
+Specification
+=============
+
+A new type of file format is defined, called a "RIM" (i.e. ``.rim``) file, so
+named to evoke the image of a wheel with the tire removed, and backronym'd to
+"Remote Installable Metadata" files. ``.rim`` files are easily derivable from
+wheels, and the process of turning a wheel into a ``.rim`` is outlined
+below. The file name format exactly matches the `wheel file naming format
+<https://packaging.python.org/en/latest/specifications/binary-distribution-format/#file-format>`__
+specification, except that RIM files use the suffix ``.rim``. This means that
+all the tags used to discriminate ``.whl`` files also distinguish between
+different ``.rim`` files, and thus can be used during dependency resolution
+steps, exactly as ``.whl`` files are today. In this respect, ``.whl`` and
+``.rim`` files are interchangeable.
+
+The content of a ``.rim`` file is *nearly* identical to ``.whl`` files,
+however ``.rim`` files MUST contain only the ``.dist-info`` directory from a
+wheel. No other top-level file or directory is allowed in the ``.rim`` zip
+file. The ``.dist-info`` directory MUST contain a single additional file in
+addition to those `allowed
+<https://packaging.python.org/en/latest/specifications/binary-distribution-format/#the-dist-info-directory>`__
+in a ``.whl`` file's ``.dist-info`` directory: a file called
+``EXTERNAL-HOSTING``.
+
+The format of this file contains lines of key-value pairs separated by a
+colon, in what :pep:`1` calls in its "PEP Header Preamble" section,
+:rfc:`2822` style headers. The following header keys are mandatory and
+exclusive (all headers are case-insensitive):
```

Comment:
```suggestion
:rfc:`2822` style headers. The following header keys are mandatory, 
should only appear once per file, and are case insensitive:
```
I think this is clearer that they should only appear once (the original text makes it sounds like the headers are mutually exclusive).

Reply:
See my comment below, re: the suggestion that this file be JSON.